kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -345,12 +404,31 @@ impl<'tcx> KaniAttributes<'tcx> {
                     };
                     harness.stubs.push(self.stub_for_relative_item(name, replacement_name));
                 }
+                KaniAttributeKind::StubVerified => {
+                    for contract in self.interpret_stub_verified_attribute() {
+                        let Ok((name, def_id, _span)) = contract else {
+                            continue;
+                        };
+                        let Some(Ok(replacement_name)) =
+                            KaniAttributes::for_item(self.tcx, def_id).replaced_with()
+                        else {
+                            self.tcx.sess.span_err(
```

Comment:
This is returning the wrong span and the message is very generic. For the following code:

```rust
fn no_contract() {}

#[kani::proof]
#[kani::stub_verified(no_contract)]
fn harness() {
    no_contract();
}
```

I get the error:
```
error: Invalid `stub_verified` attribute format
 --> invalid_stub.rs:5:1
  |
5 | fn harness() {
  | ^^^^^^^^^^^^

error: aborting due to previous error
```
The message should be pointing to the attribute definition. Also, in this case the error is that the function has no contract.

Reply:
Done. It's now pointing to both ;)